pkg/utils/utils.go

```diff
@@ -0,0 +1,48 @@
+/****************************************************************************
+ * Copyright 2022  Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+
+// Package utils //
+package utils
+
+// CompareSlices determines if two string slices are equal
+func CompareSlices(a, b []string) bool {
+	// Required in case a nil value and an empty array is given
+	isFirstValueNil := a == nil
+	isSecondValueNil := b == nil
+
+	if !isFirstValueNil && !isSecondValueNil {
+		if len(a) != len(b) {
+			return false
+		}
+		for i, v := range a {
```

Comment:
This will return false for same array values with different order. Wondering if we're ok for all use cases.

Reply:
good catch, fixing this.